libraries/RC_Channel/RC_Channel.h

```diff
@@ -162,13 +162,12 @@ class RC_Channel {
         GUIDED       =        55, // guided mode
         LOITER       =        56, // loiter mode
         FOLLOW       =        57, // follow mode
+        ZIGZAG       =        58, // zigzag mode
         // if you add something here, make sure to update the documentation of the parameter in RC_Channel.cpp!
         // also, if you add an option >255, you will need to fix duplicate_options_exist
     };
     typedef enum aux_func aux_func_t;
 
-protected:
-
     // auxillary switch handling:
```

Comment:
This should not be copied here.

Please modify the `libraries/RC_Channel/RC_Channel.h` file to move the definition into the `public:` section.

Reply:
Yes this is what I did, I moved the `enum aux_switch_pos` and `typedef enum aux_switch_pos aux_switch_pos_t` from `protected:` to `public:`. These are done in `libraries/RC_Channel/RC_Channel.h`.